.github/workflows/wheel_tests_and_release.yml

```diff
@@ -91,22 +88,29 @@ jobs:
         env:
           CIBW_BUILD: ${{ matrix.cibw_python }}-*
           CIBW_ARCHS_LINUX: ${{ matrix.cibw_arch }}
-          CIBW_SKIP: "*-musllinux_*"
-          CIBW_MANYLINUX_X86_64_IMAGE: ${{ matrix.cibw_manylinux }}
+          CIBW_MANYLINUX_AARCH64_IMAGE: manylinux2014
+          CIBW_MUSLLINUX_AARCH64_IMAGE: musllinux_1_1
+          # Should we publish 32-bit arm wheels? They are not very common
+          # and we will require QEMU with all platforms to be set up
```

Comment:
An issue that I think might come up if we decide to do this will be that we will have to let them go anyway – the upcoming `manylinux_2_28` standard in June 2024 (following the `manylinux2014` deprecation) will not support i686 wheels anyway, so it won't make sense for supporting arm32 when we will not have x86. So my opinion is that we should not do this.

Reply:
`armv7` is embedded devices, Raspberry Pi's, etc. - much more relevant than i686 I think.

But either way, I agree that we won't be doing this. At least there is no point unless NumPy does it first (and there are no plans for that).